templates/zerver/hello.html

```diff
@@ -625,6 +625,135 @@ <h2 class="float left">Leading organizations use Zulip:</h2>
+                            <p class="tour-explanation">Replying to a topic is easy: click any message, or tap &lsquo;r&rsquo; to reply to the message in blue.</p>
+                        </div>
+                    </div>
+                </div>
+                <div class="item">
+                    <div class="item-inner">
+                        <p class="tour-item-header tour-item-header-top-push">Zulip Free is free for an unlimited number of users.</p>
+                        {% if register_link_disabled %}
+                        {% elif only_sso %}
+                        <a href="{{ url('login-sso') }}">
+                            <button href="" type="button"
+                              name="button" class="call-to-action">{{ _('Log in now') }}</button>
+                        </a>
+                        {% else %}
+                        <a href="{{ url('register') }}">
+                            <button href="" type="button"
+                              name="button" class="call-to-action">{{ _('Try it free') }}</button>
+                        </a>
+                        {% endif %}
+                        <div class="other-resources">
+                            <h3>Other resources:</h3>
+                            <ul>
+                                <li><a href="/plans/">Plans and pricing</a></li>
+                                <li><a href="/features">Features</a></li>
+                                <li><a href="/why-zulip">Why Zulip</a> (10 min read)</li>
+                                <li>First time users&rsquo; guide: <a href="https://zulipchat.com/help/getting-your-organization-started-with-zulip">admin</a>, <a href="https://zulipchat.com/help/getting-started-with-zulip">user</a></li>
+                                <li><a href="https://zulip.readthedocs.io/en/stable/production/install.html">Installing Zulip on-premise</a></li>
+                            </ul>
+                        </div>
+                    </div>
+                </div>
+              </div>
+
+
+              <a class="carousel-control left" href="#myCarousel" data-slide="prev">&lsaquo;</a>
+              <a class="carousel-control right" href="#myCarousel" data-slide="next">&rsaquo;</a>
+              <!-- Carousel indicators
+                <ol class="carousel-indicators">
+                <li data-target="#myCarousel" data-slide-to="0" class="active"></li>
+                <li data-target="#myCarousel" data-slide-to="1"></li>
+                <li data-target="#myCarousel" data-slide-to="2"></li>
+                <li data-target="#myCarousel" data-slide-to="3"></li>
+                <li data-target="#myCarousel" data-slide-to="4"></li>
+                <li data-target="#myCarousel" data-slide-to="5"></li>
+                <li data-target="#myCarousel" data-slide-to="6"></li>
+              </ol>  -->
+            </div>
```

Comment:
I hackishly added bootstrap 2.3.2 to the page as part of an upgrade attempt, and it ended up looking like this: 

![image](https://user-images.githubusercontent.com/2746074/39881629-59a33fae-5436-11e8-8d01-7e9d2a93395d.png)

Is that what you had, or did my hackery not work right?

I think basically we're fine with having `hello.html` along get bootstrap from a CDN for now until we can upgrade properly (I've determined that it's definitely a lot of work to upgrade the logged-in app, because of a bunch of typeahead fixes we've done).

Reply:
Not quite - the dots are supposed to be below the lighter tour window. I'll just link to the CDN directly on the hello page for now!